vlib/v/checker/tests/option_fn_err.out

```diff
@@ -1,9 +1,9 @@
 vlib/v/checker/tests/option_fn_err.vv:40:9: error: assert can be used only with `bool` expressions, but found `bool` instead
-   38 |
+   38 | 
```

Comment:
Hmm... v fmt should've removed the blank space, shouldn't it?

Reply:
The space is not from the .vv file, but from the output of the error message.
It is produced by:
`v vlib/v/checker/tests/option_fn_err.vv 2> vlib/v/checker/tests/option_fn_err.out`

Reply:
i.e. `v fmt -verify vlib/v/checker/tests/option_fn_err.vv` passes.